### PR TITLE
Update api 1-9 docs

### PR DESCRIPTION
**Pull request by ybondur**

### CODE REVIEW

File: app/api/api_v2/entities/market.rb [hunk 1]
```diff
@@ -4,7 +4,13 @@
...
   module Entities
     class Market < Base
-      expose :id, documentation: "Unique market id. It's always in the form of xxxyyy, where xxx is the base currency code, yyy is the quote currency code, e.g. 'btcusd'. All available markets can be found at /api/v2/markets."
+      expose :id, documentation: {
+        type: String,
+        description: "Unique market id. It's always in the form of xxxyyy,"\
+              "where xxx is the base currency code, yyy is the quote"\
+              "currency code, e.g. 'btcusd'. All available markets can"\
+              "be found at /api/v2/markets."
+      }
```
Comment (mnaichuk) [inline app/api/api_v2/entities/market.rb]: Also add other fields like in https://github.com/rubykube/peatio/blob/master/docs/api/peatio_user_api_v2.md#market

File: app/api/api_v2/fees.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Fees < Grape::API
     DepositFee  = Struct.new(:currency, :type, :fee)
     TradingFee  = Struct.new(:market, :ask_fee, :bid_fee)
 
-    desc 'Returns withdraw fees for currencies.'
+    desc 'Returns withdraw fees for currencies.', deprecated: true
```
Comment (mnaichuk) [inline app/api/api_v2/fees.rb]: We wouldn't delete this APIs in 1-9 so no need for `deprecated: true`

File: app/api/api_v2/orders.rb [hunk 3]
```diff
@@ -8,87 +7,85 @@ class Orders < Grape::API
...
-    get "/order" do
+    get '/order' do
       order = current_user.orders.where(id: params[:id]).first
       raise OrderNotFoundError, params[:id] unless order
+
       present order, with: APIv2::Entities::Order, type: :full
     end
 
-    desc 'Create multiple sell/buy orders.', scopes: %w(trade)
+    desc 'Create multiple sell/buy orders.', scopes: %w[trade], success: APIv2::Entities::Order
```
Comment (mnaichuk) [inline app/api/api_v2/orders.rb]: is_array: true

File: app/api/api_v2/orders.rb [hunk 4]
```diff
@@ -8,87 +7,85 @@ class Orders < Grape::API
...
+    post '/order/delete' do
+      order = current_user.orders.find(params[:id])
+      Ordering.new(order).cancel
+      present order, with: APIv2::Entities::Order
+    rescue StandardError
+      raise CancelOrderError, $ERROR_INFO
     end
 
-    desc 'Cancel all my orders.', scopes: %w(trade)
+    desc 'Cancel all my orders.', scopes: %w[trade], success: APIv2::Entities::Order
```
Comment (mnaichuk) [inline app/api/api_v2/orders.rb]: is_array: true

**Review by mnaichuk — CHANGES_REQUESTED** [review]

**Review by ysv — APPROVED** [review]

**Pull request merged**